ingestion/src/metadata/profiler/profiler/sampler.py

```diff
@@ -40,6 +42,27 @@
 RANDOM_LABEL = "random"
 
 
+def _object_value_for_elem(self, elem):
+    try:
+        return self._object_lookup.get(elem, elem)
+    except KeyError as err:
+        util.raise_(
+            LookupError(
+                "'%s' is not among the defined enum values. "
+                "Enum name: %s. Possible values: %s"
+                % (
+                    elem,
+                    self.name,
+                    util.langhelpers.repr_tuple_names(self.enums),
+                )
+            ),
+            replace_context=err,
+        )
+
+
```

Comment:
If we are doing a get in dict there is no possibility of getting a key error so no need sof this exception handling!

Reply:
Yes, we can remove that.